### ISSUE TITLE
Develop a unit test-like physics "testbed" system

### ISSUE THREAD

**Issue by nusbaume** [ESCOMP/CAM-SIMA#214]
### What is the feature/what would you like to discuss?

CAMDEN would benefit from a testing infrastructure that would allow it to cheaply test all of its relevant CCPP physics schemes to ensure that they run as expected, and produce the expected results.  This can be done using the "snapshot" feature from CAM and the null dycore in CAMDEN, along with the CIME regression-testing infrastructure.  A google doc describing the overall goal, design objectives, and planned workflows for adding and running these physics tests can be found here:

https://docs.google.com/document/d/1BoVsjOzmxc7lT7rAhovDeQzi0cpxhjFeZPrm5NVgqKk/edit

I should note that this issue is mostly to just get the test system working on a user's machine.  Once that is done we can see about moving it to Github Actions, which will likely be a separate issue.

### Is there anyone in particular you want to be part of this conversation?

@cacraigucar @peverwhee 

Please feel free to look over the document and let me know if you have any questions or concerns.  Thanks!

### Will this change (regression test) answers?

No

### Will you be implementing this enhancement yourself?

Yes

**Comment by nusbaume** [comment]
This issue has been superseded by issue #236.
